app/Models/Beatmapset.php

```diff
@@ -1088,6 +1088,16 @@ private function getBBCode()
         return new BBCodeFromDB($description, $post->bbcode_uid, $options);
     }
 
+    public function getDisplayTitle()
+    {
+        $showOriginal = auth()->user()->userProfileCustomization->options['beatmapset_title_show_original'] ?? false;
```

Comment:
I just remembered about title in notification. I think that should follow this setting as well?

(And thus this function probably should accept argument `$user` and use it instead. Accessing session user within model seems bad)

Also the attribute can be accessed by `...Customization->beatmapset_title_show_original`.

Going a bit further, the `userProfileCustomization` can use `$user->profileCustomization()` instead to be guaranteed to be not null. And get the default directly from the relevant model.

Reply:
`profileCustomization`'s conditional insert sounds like a terrible idea with notifications for other users

Reply:
Apparently it's already being called when loading the page itself ( ﾟ ヮﾟ)

Reply:
which has nothing to do with the user receiving the notification where the setting is going to be on?

Reply:
hmm maybe just update it to stop creating a new entry. Still weird defining default for same thing in two different places.

Reply:
I wonder if it's better to include `title_unicode` in the notification instead, otherwise, the previous notifications would be displaying based on whatever the setting was when it was sent instead of what the current setting is (and being completely ignored for the ones already without the extra data)

Reply:
sure, if you can figure out how to do it with mail notification :upside_down_face: 

(although that sounds a bit like waste of space)